samples/run/README.md

```diff
@@ -0,0 +1,96 @@
+# Data Validation on Cloud Run
+
+### Build Docker Image
+
+You will need to build a Docker image to be used by Cloud Run. In order to add
+Teradata or Oracle, you will need to customize the Dockerfile and add your
+licensed utilities.
+
+```
+export PROJECT_ID=<PROJECT-ID>
+gcloud builds submit --tag gcr.io/${PROJECT_ID}/data-validation \
+    --project=${PROJECT_ID}
+```
+
+### Deploy to Cloud Run
+
+```
+gcloud run deploy --image gcr.io/${PROJECT_ID}/data-validation \
+    --project=${PROJECT_ID}
+```
+
+### Test Cloud Run Endpoint
+
+You can easily run a request via Python. For example:
```

Comment:
We can mention here that this is the `test.py` code. We should also add that the service takes in a JSON configuration object which is equivalent to the ConfigManager._config dictionary.